src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -47,6 +47,13 @@ internal class WinHttpHandler : HttpMessageHandler
     public class WinHttpHandler : HttpMessageHandler
 #endif
     {
+#if NET46
+        public static Version HttpVersion20 => new Version(2,0);
+        public static Version HttpVersionUnknown => new Version(0,0);
+#else
+        public static Version HttpVersion20 => HttpVersion.Version20;
+        public static Version HttpVersionUnknown => HttpVersion.Unknown;
```

Comment:
oh...good point. although they aren't exposed in the contract, I'll change them to `internal`